wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -321,6 +321,12 @@ private void createNotificationChannels() {
                 R.string.notification_synchronization_channel_name,
                 R.string.notification_synchronization_channel_description,
                 NotificationManager.IMPORTANCE_LOW);
+
+        // Push notifications
+        createNotificationChannel(getString(R.string.fcm_notification_channel_id),
```

Comment:
Since the push notifications channel needs to be referenced from the manifest, we have to store it in `strings`